pkg/dashboard/store/metrics/metrics.go

```diff
@@ -18,19 +18,18 @@ package metrics
 import (
 	"context"
 
+	"github.com/go-logr/logr"
 	"github.com/prometheus/client_golang/prometheus"
 	"go.uber.org/fx"
-	ctrl "sigs.k8s.io/controller-runtime"
 
 	"github.com/chaos-mesh/chaos-mesh/pkg/dashboard/core"
 )
 
 const chaosDashboardSubsystem = "chaos_dashboard"
 
-var log = ctrl.Log.WithName("metrics-collector")
-
 // Collector implements prometheus.Collector interface
 type Collector struct {
+	log             logr.Logger
```

Comment:
```suggestion
	Log             logr.Logger
```

It should use exported field with `fx` dependency injection.

rest LGTM! You are almost there.